### ISSUE TITLE
FAQ page - update styling

### ISSUE THREAD

**Issue by minayao** [CollActionteam/CollAction#48]

**Comment by efel85** [comment]
The FAQ page needs to give visitors easy access to any questions and answers they might have. Fast overview of all questions and easy access to answers is essential, in form of dropdown texts or anchors.

**Comment by ronvdakker** [comment]
Frequently asked questions
 
WHAT IS COLLACTION?
CollAction is a not-for-profit organization based in the Netherlands that helps people to solve Collective Action Problems through crowdacting. 

HUH? WHAT IS CROWDACTING? 
The term Crowdacting, as it is used here, is a term that has been introduced by CollAction. Basically, it means: ‘taking action collectively and conditionally with the purpose of achieving positive impact on the world and its inhabitants’.
Or, in simpler terms: With crowdacting we ask: ‘Would you take action if you knew that a hundred/a thousand/a million/a billion others would do so too?’ An example of a crowdacting project: ‘If we can find 10,000 people that commit to switch to renewable energy if 9,999 others would do the same, we will all switch together’. The concept is very similar to crowdfunding, but people commit to actions, instead of money.

In practice, a crowdacting project follows three steps: 
1.	The ProjectStarter launches a project on collaction.org. He/she proposes a certain action (for example switching to renewable energy), a target (minimum number of participants), and a deadline. 
2.	ProjectSupporters commit to take the action, if the target is met within the deadline. 
3.	If and when the target is met, everybody acts collectively. 

More information on crowdacting can be found on www.crowdacting.org.

CROWDACTING. IS THAT NEW?
Although the term crowdacting in its current meaning is new, the underlying concept is not. There are plenty of other examples of collective actions that contain elements of crowdacting. Think for instance of boycotts, demonstrations, collective bargaining initiatives, or petition websites. The difference is the fact that crowdacting combines three elements: 1) It’s (explicitly) conditional (it only happens if a set target is met); 2) it’s for social and/or ecological good; and 3) the action goes beyond signing a petition.
More information on how crowdacting resembles and differs from other initiatives can be found on www.crowdacting.org.

WHAT ARE COLLECTIVE ACTION PROBLEMS?
Collective action problems are situations where it is in the interest of the group to work together, but in which coordination is a challenge. Because of this, every individual has an incentive to act in their own interest. This video explains it clearly.

A well-known example of a collective action problem is the tragedy of the commons.
The commons is a communal piece of meadow grounds where cows of different farmers can graze. Every farmer wants to earn as much as possible against the lowest possible costs and therefore has the individual incentive to let as many of his cows graze on the communal grass as possible. However, other farmers will do the same and soon the commons will be destroyed due to overgrazing. There is little point to do something about this for each individual farmer – knowing (or fearing) that other farmers will try and use the largest possible share of the communal grass and the commons will be grazed as much as possible anyway. The farmer is better off adding some more cows, so he will, at least, benefit from it in the short term. However, in the long term, all the farmers and as a consequence the community will lose out, because the piece of land can be written off due to overgrazing. 
 
Examples of (serious and less serious) collective action problems are: global warming, traffic jams, overpopulation, wearing of uncomfortable high heels, dismal working conditions, keeping a student house clean, corruption, overfishing, etcetera.  
 
Traditionally, there are two solutions to collective action problems: 
1.	Privatisation: the commons can be divided between the farmers. Now the farmers benefit from taking good care of their own piece of land, so that it is of use to them in the long term. 
2.	Regulation: Agreements can be made and enforced about the usage of the commons. It is, for example, possible to decide on a maximum number of cows per farmer.

However, these solutions are not applicable to all collective action problems. It is not so easy (or desirable) to privatise certain goods (how do you privatise the ozone layer? Or migrating fish populations?). Regulating and enforcing (cross border) problems has proven to be difficult - just think about the cumbersome process to do something about climate change on an international level. 
So, we need a new solution to old problems: crowdacting.

WHERE CAN I LEARN MORE ABOUT CROWDACTING?
www.crowdacting.org
CAN I START A PROJECT MYSELF?
Certainly! You can start a project here. Make sure to check if your idea meets the CollAction criteria.

WHAT ARE THE CRITERIA THAT A PROJECT HAS TO MEET?
A project can only be listed when it meets the following criteria:
•	The goal of the project is to make a positive societal or ecological contribution to your neighbourhood, country or the world. 
•	The project is not geared towards personal gain. 
•	The project does not include activities that are focussed on conversion or activism (religious or political). 
•	The project does not include activities that are illegal or do not abide by the official legislation of the Netherlands or the country of implementation.  
•	The online project registration form is completed fully and truthfully and has a clear and easily readable project description and goal. The CollAction team can ask for clarification and/or edit your text if necessary. 
•	The ProjectStarter has thought through how people can be moved from commitment to action. We can help you with this! 
•	The project is ambitious but realistic – the CollAction evaluation commission judges if this is the case. The ProjectStarter can activate his/her own network, and/or has a good plan to achieve the target.
•	The ProjectStarter commits to measuring the impact of the action (e.g. the number of people that have acted as a result of the project) and to sharing this information with CollAction.

HOW LONG WILL AN ACTION LAST?
Some actions are a one-off– think of the switch to a fair bank or green energy. Other projects can have a longer duration, such as eating less meat, periodically visiting lonely elderly people or helping refugees with their Dutch lessons. In other words, the duration of an action differs from one project to the other. 

WHAT HAPPENS WHEN A TARGET HAS NOT BEEN REACHED?
If the target is not met by the deadline, no one needs to act. Crowdacting means: No cure, no action. However, if you are really inspired to take action anyway, we will of course applaud this. 

HOW DO I KNOW THAT PEOPLE WILL REALLY TAKE ACTION?
You can never be 100% sure. In the first place, crowdacting is about trust. Trusting that people want to work together to create a better world. Trusting that people are attached to their personal ideals. And trusting that people will keep their promises. We hope to build a community of people that take their commitments seriously and try to create a culture in which agreement=agreement.  
That being said, we will incorporate certain processes and tools to ensure that as many project supporters proceed to taking action. What these processes are depends on the type of project. For some projects (such as switching projects, for example to a fair bank or green energy) the switch can already be started conditionally at the time of commitment. When the target is reached, supporters of the project will automatically be transferred to the fair bank or renewable energy provider. When it comes to local projects, social control will play a bigger role. Other projects might be purely based on trust. We will be testing different tools and approaches in the upcoming projects. We ask ProjectStarters to seriously consider this question  – the team of CollAction can help you think this through!
